azure-iot-device/tests/common/pipeline/test_pipeline_stages_base.py

```diff
@@ -3586,9 +3677,8 @@ def test_completes_waiting_connect_ops_on_arbitrary_exception(
             assert op.original_callback.call_count == 1
             assert op.original_callback.call_args == mocker.call(op=op, error=arbitrary_exception)
 
-    @pytest.mark.parametrize("state", [pipeline_stages_base.ReconnectState.NEVER_CONNECTED])
     @pytest.mark.it(
-        "Completes all waiting ops with the transient failure if the connection fails with a transient error"
+        "Completes all waiting ops with the transient failure if a first-time connection fails with a transient error"
```

Comment:
>transient error [](start = 102, length = 15)

for a first connection case , does this matter if it is transient error or it includes transient and non transient error of all kinds ?

Reply:
That's correct.  The tests for permanent (arbitrary, non-transient) errors are above.

---
In reply to: [437760479](https://github.com/Azure/azure-iot-sdk-python/pull/576#discussion_r437760479) [](ancestors = 437760479)